### DISCUSSION TITLE
calculation error

### DISCUSSION THREAD

**Discussion by hekungeo** [Anura3D/Anura3D_OpenSource#7] [Q&A] [answered]
hi,

I was doing the slope failure simulation using Anura3D_2022. However, some problems occurred during the calculation. As following:

**Error: Did not find local coordinates of particle in 1 within limit number of iterations.**

![image](https://user-images.githubusercontent.com/105496180/170233693-de7cf734-1266-4293-8fd4-b8087fdc696c.png)

and my mesh is shown like this:
![image](https://user-images.githubusercontent.com/105496180/170233887-9af324d4-30b0-48c0-a1ff-b462d5ff9162.png)

Could you please help me with this?

Thx in advance.

**Comment by FrancescaCeccato** [comment, marked as answer]
Usually this error means that the algorithm cannot find a new element for a moving material point. This can be because the point is moving too fast, so maybe there is an instability issue, or loading is not correct, or boundary conditions are not correctly applied. However, in this case it is happening at the fist time step, so I think there must be an error in the geometry and mesh, maybe you have duplicate entities. Try to regenerate your model from scratch.

**Reply by hekungeo** [reply]
Hi Francesca,

Thank you for your reply. Once I use point coordinates instead of dxf file to generate the model, it works well. Thank you
